vcr.yml

```diff
@@ -6,14 +6,18 @@ instance:
   region: aws.euw1
   entrypoint: [yarn, run-server]
   application-id: <APP_ID>
```

Comment:
does this line need to be here? looks like we use the --app-id arg in the deploy script